recipes/rosco/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "rosco" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+# source:
+#   url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
+#   sha256: 3e907f7d2f8369cb20c08050985e00473176e8b5cde85ebda8f77e8d18135a2f
+#   # Download the tar-ball and do `openssl sha256 file.tar.gz` to update
+
+source:
+  git_url: https://github.com/NREL/ROSCO.git
+  # git_sh: 47a13446063a136c7f22ea09d3569280b62668dd
+  git_rev: master
```

Comment:
Please build from a source tarball, not git

Reply:
And done!